.github/workflows/build-images-beta.yaml

```diff
@@ -0,0 +1,138 @@
+name: Beta Image Release Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Docker Image Tag'
+        required: true
```

Comment:
I'm fine with this trigger but I'm wondering why it differs from the trigger used for the hotfix images? That is, why don't we also use `workflow_dispatch` for hotfix images (or the other way around)?

Reply:
@pchaigno yep. I noticed the same. I think the `workflow_dispatch` was not available by the time the hotfix workflow was created.

Reply:
Is there an advantage to one compared to the other?

Reply:
I don't see any, no :thinking:

Reply:
Ok. Then probably not worth overthinking this :slightly_smiling_face: